### ISSUE TITLE
Command Prioritization

### ISSUE THREAD

**Issue by ThandiBrown** [talonvoice/talon#241]
```
at <phrase> and <phrase>:
    "["
    insert(user.formatted_text(phrase_1, "SNAKE_CASE"))
    "]["
    insert(user.formatted_text(phrase_2, "SNAKE_CASE"))
    "]"
and:" and "
```
yes so it chooses to pick the second/shorter command
If I say:
`"at flex and yank"`
It will output:
`[flex] and y`
Even though the command history registered it as:
`"at flex and yank"`
(I'm saying the phrase pretty quickly)
The desired output is:
`[flex][yank]`

**Comment by rntz** [comment]
I've been told to put this here.

I'm trying to have a command `new line` or `enter` in dictation mode that produces a new line, but `escape new line` or `escape enter` will say "new line" or "enter". So I tried:

```
mode: dictation
-
<phrase>: "{phrase}"
^escape <phrase>$: "ESCAPE {phrase}"
new line: "NEW LINE"
enter: "ENTER"
```

with the above, saying "escape enter" will produce "ESCAPE enter" as desired, but "escape new line" will produce "escapeNEW LINE" :(

**Comment by lunixbochs** [comment]
I couldn't repro the first one initially, but after talking in Slack it turns out there's an additional command mapped:
```
at <phrase>:
    "["
    insert(user.formatted_text(phrase, "SNAKE_CASE"))
    "]"
```

@rntz do you have any other dictation commands mapped or is your script set pretty much just this?

**Comment by rntz** [comment]
I have many more commands mapped in dictation mode, but I tried making a new mode where I have basically only those commands mapped, and the behavior is the same.

```
mode: user.bug
-
<phrase>: "{phrase}"
^escape <phrase>$: "ESCAPE {phrase}"
new line: "NEW LINE"
magic: "ENTER"
^bug mode quit | command mode$:
  mode.disable("user.bug")
  mode.enable("command")
```

**Comment by rntz** [comment]
And on gen2, it usually produces an at sign: `escape magic` -> `ESCAPE @magic`

**Comment by lunixbochs** [comment]
All of these are fixed now in the latest beta (v0.1.4-179)
